app/assets/javascripts/angular_apps/authentication/controllers/sign_in.controller.js

```diff
@@ -17,8 +17,14 @@
     $scope.signIn = function() {
       // Code to use 'angular-devise' component
       Auth.login($scope.credentials).then(function(user) {
-        $window.location.href = '/dashboard#/';
-        alert('Successfully signed in user!')
+        if(user.type == 'admin'){
+          console.log('Hi Admin');
+          $window.location.href = 'admin#/';
+          alert('ADMIN Successfully signed in!')
+        } else{
+          $window.location.href = '/dashboard#/';
+          alert('Successfully signed in user!')
+        };
```

Comment:
after login in u did not handle a single case for the admin or the normal user to be authenticated on there respective pages. i.e Current admin can access all normal user pages and normal user can access admin pages.